apps/labs/pages/api/annual-report.pdf.ts

```diff
@@ -0,0 +1,33 @@
+import type { NextRequest } from 'next/server';
+
+export const config = {
+  runtime: 'experimental-edge',
+};
+
+export default async function handler(req: NextRequest) {
+  // TODO - remove after checking the logs on Vercel
+  console.log(`
+  req.url ${req.url}
+  req.referer ${req.headers.get('referer')}
+  req.ip ${req.ip}
```

Comment:
This is the value I am most curious to check in the Vercel logs because it was turning up empty for me when testing locally (which I think is to be expected but I'm not 100% sure).